policies/submissions.md

```diff
@@ -1,49 +1,53 @@
 - Organizers and judges reserve the right to disqualify entries and winners who violate the [Code of Conduct](/code-of-conduct.html) and [Rules](/policies/rules.html) outlined on this website
 
-## Weekly Prize Submission
+**Ensure that you have also read and understood the [full rules!](rules.html)**
+
+# Submission Details
+
+## Deliverables
+
+The following are all **required** unless stated otherwise.
+
+1. URL to your working app (the website, or a TestFlight/Google Play opt-in link).
+1. URL to a screenshot we can share with voters and the media.
+1. The [CODEVID-19 problem](rules.html#the-problems) you're addressing.
+1. Short description/pitch for your app we can share on Twitter and social media (140 characters or less)
+1. Long description/pitch for your app we can use to describe your app to voters, judges, and the media.
+1. Instructions for interacting with your app (if not included in the app itself).
+1. An email address that judges can contact should they have questions or concerns. This email will **not** be shared publicly.
+1. URL to your project on [FindCollabs](https://findcollabs.com/hackathon/codevid-19-isp21fkqtjupchx7kjed)
+1. **(Optional)** For open-source projects, we encourage a link to the source code repository (e.g. GitHub, GitLab, Bitbucket, etc.).
+
+### Weekly Prize Submission
 
 - Your submission does not need to be complete, but you must be able to demonstrate it.
 - Teams may submit their projects for multiple weekly considerations; however, new progress must be demonstrated leading up to each submission.
+- Be sure to review the [Deliverables](#deliverables) and have them ready before we close submissions!
 
-## Final Prize Submission
+### Final Prize Submission
 
 When submitting for final consideration, teams must also include:
 
-1. A link to a short, 3 minutes or less, pitch and demo that clearly explains:
-   a. the problem being solved
-   b. how to use the app
-   c. any unique features or information pertinent to the judging
-
+- A link to a short pitch and demo (3 minutes or less) that clearly explains:
+  1. The problem being solved
+  1. How to use the app
+  1. Any unique features or information pertinent to the judging
 - When submitting projects for final consideration, please ensure that the team is available during the judging period to answer questions about your project.
 - During final judging some teams may be asked to meet with the judges via video conference. Please ensure some or all team members are available.
-- Failure to communicate and answer questions from the judges may disadvantage your place in the competition, and could disqualify you from prizes.
+- Failure to communicate and answer questions from the judges may impact your project's placement in the competition and could disqualify you from prizes.
```

Comment:
I like the change to `project's placement` but let's keep the word `disadvantage` as `impact` is more obtuse.